framework/quota/src/main/java/org/apache/cloudstack/quota/constant/QuotaTypes.java

```diff
@@ -55,6 +55,7 @@ public class QuotaTypes extends UsageTypes {
         quotaTypeList.put(VOLUME_SECONDARY, new QuotaTypes(VOLUME_SECONDARY, "VOLUME_SECONDARY", UsageUnitTypes.GB_MONTH.toString(), "Volume secondary storage usage"));
         quotaTypeList.put(VM_SNAPSHOT_ON_PRIMARY, new QuotaTypes(VM_SNAPSHOT_ON_PRIMARY, "VM_SNAPSHOT_ON_PRIMARY", UsageUnitTypes.GB_MONTH.toString(), "VM Snapshot primary storage usage"));
         quotaTypeList.put(BACKUP, new QuotaTypes(BACKUP, "BACKUP", UsageUnitTypes.GB_MONTH.toString(), "Backup storage usage"));
+        quotaTypeList.put(NETWORK, new QuotaTypes(NETWORK, "NETWORK", UsageUnitTypes.GB_MONTH.toString(), "Network usage"));
```

Comment:
@SadiJr, the type `GB_MONTH` does not seem to make sense here; I think `COMPUTE_MONTH` fits better.

Reply:
Done, thanks.